fri/src/prover/mod.rs

```diff
@@ -103,6 +104,11 @@ struct FriLayer<B: StarkField, E: FieldElement<BaseField = B>, H: Hasher> {
     _base_field: PhantomData<B>,
 }
 
+struct FriLastLayer<B: StarkField, E: FieldElement<BaseField = B>> {
+    evaluations: Vec<E>,
+    _base_field: PhantomData<B>,
+}
```

Comment:
Do we actually need `_base_field` here? Maybe we can simplify this as:
```Rust
struct FriLastLayer<E: FieldElement>(Vec<E>);
```
Also, I wonder if we should call it just `FriRemainder`?

Reply:
Sounds good, I use `remainder` throughout the code now instead.